bundles/org.openhab.binding.renault/src/main/resources/OH-INF/i18n/renault.properties

```diff
@@ -79,7 +79,7 @@ channel-type.renault.chargingstatus.state.option.UNKNOWN = Unknown
 channel-type.renault.estimatedrange.label = Estimated Range
 channel-type.renault.estimatedrange.description = Estimated range of the car.
 channel-type.renault.externaltemperature.label = External Temperature
-channel-type.renault.externaltemperature.description = Temperature outside of the car
+channel-type.renault.externaltemperature.description = Temperature outside of the car.
```

Comment:
Should probably not have mentioned this, as it would need to be updated in the XML as well. 😉